pom.xml

```diff
@@ -136,6 +177,9 @@
                             <links>
                                 <link>http://java.sun.com/javase/6/docs/api/</link>
                             </links>
+                            <sourcepath>src/main/java;target/generated-sources/swagger/src/main/java</sourcepath>
+                            <excludePackageNames>io.swagger.client.*</excludePackageNames>
```

Comment:
This configuration used when we publish the release to public maven repo. Not for javadoc we publish on Github. I am not sure if we should use it.